build/testBuildScriptGenerator.sh

```diff
@@ -29,14 +29,6 @@ dotnet test \
     --logger:"xunit;LogFilePath=$ARTIFACTS_DIR\testResults\\$testProjectName.xml" \
     -c $BUILD_CONFIGURATION
 
-testProjectName="Detector.Tests"
-cd "$TESTS_SRC_DIR/$testProjectName"
-dotnet test \
-    --blame \
-    --test-adapter-path:. \
-    --logger:"xunit;LogFilePath=$ARTIFACTS_DIR\testResults\\$testProjectName.xml" \
-    -c $BUILD_CONFIGURATION
```

Comment:
A quick question, this will be included after we integrated library?

Reply:
Ideally we want Detector and Detector.tests projects to be built, tested and released separately (you can think of them as though they are part of a different repo). So build script generator will be referencing Detector via nuget package reference.
To summarize, once we integrate Detector library, we will no longer require this script as it is being tested as part of `testDetector.sh` file.